### PR TITLE
[MWAR-371] - Overlays break first-win rule for web resource with target path ending with '/'

### PR DESCRIPTION
**Pull request by W-Alphonse**
[MWAR-371] - Overlays break first-win rule for web resource with target path ending with '/'

### CODE REVIEW

File: src/it/MWAR-371/custom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+			<type>war</type>
+		</dependency>
+	</dependencies>
+	<build>
+
+		<plugins>
+			<plugin>
+				<artifactId>maven-war-plugin</artifactId>
+				<version>3.2.3-SNAPSHOT</version>
+				<configuration>
```
Comment (khmarbaise) [inline src/it/MWAR-371/custom/pom.xml]: The hard coded version is not a good idea...you should look at the other IT's as an example how to do this correctly. `@project.version@` ....

File: src/it/MWAR-371/pom.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		<module>generic</module>
+		<module>custom</module>
+	</modules>
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<artifactId>maven-war-plugin</artifactId>
+					<version>2.6</version>
+					<configuration>
```
Comment (khmarbaise) [inline src/it/MWAR-371/pom.xml]: Why defining an different version than you are build with this build?

**Comment by rfscholte** [comment]
PR cannot be merged due to removed code. Can you update the patch?

**Comment by rfscholte** [comment]
@W-Alphonse  PR cannot be merged anymore, please provide a new one.

**Comment by jira-importer** [comment]
Resolve #453

**Pull request closed without merging**